client/app/recommendationListEntry.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import styles from './recommendationListEntry.css';
 
-const RecommendationListEntry = ({ rec }) => (
-  <div className="entry">
-    <img class={styles.img} src={rec.photo_url} alt="Recommended Places"></img>
-    <div>{rec.title}</div>
-    <div>{rec.description}</div>
-    <div>{rec.price}</div>
-  </div>
-);
+const RecommendationListEntry = ({ row }) => {
+
+  return (
+    <div className={styles.divRow}>{row.map(item => (
```

Comment:
```RecommendationListEntry``` component seems to correspond to a row of entries here, rather than a single entry. The array mapping logic may make more sense in the ```RecommendationList``` component.